src/atsamd/adc.c

```diff
@@ -12,7 +12,29 @@
 #define ADC_TEMPERATURE_PIN 0xfe
 DECL_ENUMERATION("pin", "ADC_TEMPERATURE", ADC_TEMPERATURE_PIN);
 
-#if CONFIG_MACH_SAMD21
+#if CONFIG_MACH_SAMC21
+
+#define ADC_INPUTCTRL_MUXNEG_GND 0x18
+
+#define SAMD51_ADC_SYNC(ADC, BIT)
+static const uint8_t adc_pins[] = {
+    /* ADC0 */
+    GPIO('A', 2), GPIO('A', 3), GPIO('B', 8), GPIO('B', 9), GPIO('A', 4),
+    GPIO('A', 5), GPIO('A', 6), GPIO('A', 7), GPIO('A', 8), GPIO('A', 9),
+    GPIO('A', 10), GPIO('A', 11),
+    /* ADC1 */
+    GPIO('B', 0), GPIO('B', 1), GPIO('B', 2), GPIO('B', 3), GPIO('B', 8),
+    GPIO('B', 9), GPIO('B', 4), GPIO('B', 5), GPIO('B', 6), GPIO('B', 7),
+    GPIO('A', 8), GPIO('A', 9)
+};
+static const uint8_t sdadc_pins[] = {
+    /* AINN */
+    GPIO('A', 6), GPIO('B', 8), GPIO('B', 6),
+    /* AINP */
+    GPIO('A', 7), GPIO('B', 9), GPIO('B', 7)
+};
```

Comment:
Dead code?

Reply:
Oh, whoops - thought I got rid of that for this PR.